src/util/address.rs

```diff
@@ -409,10 +410,7 @@ impl Payload {
     /// Creates a pay to (compressed) public key hash payload from a public key
     #[inline]
     pub fn p2pkh(pk: &ecdsa::PublicKey) -> Payload {
-        let mut hash_engine = PubkeyHash::engine();
-        pk.write_into(&mut hash_engine)
-            .expect("engines don't error");
-        Payload::PubkeyHash(PubkeyHash::from_engine(hash_engine))
+        Payload::PubkeyHash(PubkeyHash::hash(&pk.serialize()))
```

Comment:
nit: There is a `to_bytes()` method for doing this. I don't like using psbt specific utilities for this.

Reply:
Done!